### PR TITLE
Website using  angular/react framework

### PR DESCRIPTION
**Pull request by Shibon-ll**
### Requirements

* Website to support SPA App using Angular/React 

### Description of the Change
Added Condition to check out react/angular template
Code to validate angular and react template
Code to build angular/react template

### Benefits

User can now able to create website using angular /react framework

### Possible Drawbacks

Nil

### Applicable Issues

Nil

### CODE REVIEW

File: builds/jenkins-build-pack-website/Jenkinsfile [hunk 1]
```diff
@@ -134,7 +134,13 @@ node()  {
 
 				echo "create_cloudfront_url====" + create_cloudfront_url
 				send_status_email('STARTED', '')
-				validateTemplate()
+				if(service_config['runtime'] == 'angular') {
```
Comment (chimpuz) [inline builds/jenkins-build-pack-website/Jenkinsfile]: We should define a new key "framework" applicable to websites. Lets store this in catalog (dynamodb table:jazz_services) in metadata.

File: core/jazz_create-serverless-service/index.js [hunk 2]
```diff
@@ -261,6 +261,9 @@ var getServiceData = (service_creation_data, authToken, configData) => {
             var create_cloudfront_url = "true";
             serviceMetadataObj.create_cloudfront_url = create_cloudfront_url;
             inputs.RUNTIME = 'n/a';
+            if (service_creation_data.runtime == 'angular' || service_creation_data.runtime == 'react') {
```
Comment (chimpuz) [inline core/jazz_create-serverless-service/index.js]: This should be the new parameter - framework

File: builds/service-onboarding-build-pack/Jenkinsfile [hunk 3]
```diff
@@ -66,7 +66,7 @@ node  {
 
 		events.sendStartedEvent("VALIDATE_INPUT", 'input validation starts', context_map)
 		if (service_config['type'] == "api" || service_config['type'] == "function" || service_config['type'] == "website") {
-			if (service_config['runtime'].startsWith("nodejs") || service_config['runtime'].startsWith("python") ||  service_config['runtime'].startsWith("java") ||  service_config['runtime'].startsWith("go") || service_config['runtime'] == "n/a") {
+			if (service_config['runtime'].startsWith("nodejs") || service_config['runtime'].startsWith("python") ||  service_config['runtime'].startsWith("java") ||  service_config['runtime'].startsWith("go") || service_config['runtime'] == "n/a" || service_config['runtime'] == 'angular' || service_config['runtime'] == 'react') {
```
Comment (chimpuz) [inline builds/service-onboarding-build-pack/Jenkinsfile]: Please use framework paramter.

**Comment by Shibon-ll** [comment]
@chimpuz suggested changes have been made

**Pull request merged**